.github/CONTRIBUTING.md

```diff
@@ -102,6 +102,65 @@ consider:
    - first checkout your main: `git checkout main`
    - then run: `git reset --hard upstream/main`
 
+## Developing for the validator
+
+When proposing a feature or bug fix, you must decide what branch you will target:
+`main` or `dev`.
+In most cases, you will want `main`, but read below to understand the purposes of
+the two branches.
+
+### Branching policy
+
+The BIDS Validator's `main` branch tracks the most recent release version of the specification:
+<https://bids-specification.readthedocs.io/en/stable/>.
+Pull requests made against the `main` branch should implement features or fix bugs in a way
+that preserves consistency with the stable specification.
+
+The `dev` branch is for features that are not yet in the released version of the BIDS
```

Comment:
Thats silly of me to ask. They should go into their own branch since they'll be tracking a different version of the specification schema. And once the in progress BEP is merged into the specification, the validator BEP branch can then be merged into `dev`.

Reply:
I was thinking that for BEPs, we would create a `bep0XX` branch and we can even publish it to `bids-validator/bep0XX`. But I didn't want to write it out just yet.